pages/offres/[offre_id].tsx

```diff
@@ -0,0 +1,339 @@
+                    <li key={competence} className={liStyle}>
+                      {competence}
+                    </li>
+                  ))}
+                </ul>
+              </dd>
+            </>
+          )}
+
+          {offre.competencesProfessionnelles.length > 0 && (
+            <>
+              <dt className={dtStyle}>Savoir être professionnel</dt>
+              <dd className={ddStyle}>
+                <ul className={ulStyle}>
+                  {offre.competencesProfessionnelles.map((competencePro) => (
+                    <li key={competencePro} className={liStyle}>
+                      {competencePro}
+                    </li>
+                  ))}
+                </ul>
+              </dd>
+            </>
+          )}
+
+          {offre.formations.length > 0 && (
+            <>
+              <dt className={dtStyle}>Formation</dt>
+              <dd className={ddStyle}>
+                <ul className={ulStyle}>
+                  {offre.formations.map((formation) => (
+                    <li key={formation} className={liStyle}>
+                      {formation}
+                    </li>
+                  ))}
+                </ul>
+              </dd>
+            </>
+          )}
+
+          {offre.langues.length > 0 && (
+            <>
+              <dt className={dtStyle}>Langue</dt>
+              <dd className={ddStyle}>
+                <ul className={ulStyle}>
+                  {offre.langues.map((langue) => (
+                    <li key={langue} className={liStyle}>
+                      {langue}
+                    </li>
+                  ))}
+                </ul>
+              </dd>
+            </>
+          )}
+
+          {offre.permis.length > 0 && (
+            <>
+              <dt className={dtStyle}>Permis</dt>
+              <dd className={ddStyle}>
+                <ul className={ulStyle}>
+                  {offre.permis.map((permis) => (
+                    <li key={permis} className={liStyle}>
+                      {permis}
+                    </li>
+                  ))}
+                </ul>
+              </dd>
+            </>
+          )}
+        </dl>
+      </section>
+
+      {offre.infoEntreprise && (
+        <section
+          aria-labelledby='heading-entreprise'
+          className={infoEntrepriseSrOnly ? 'sr-only' : 'mt-6'}
+        >
+          <h3 id='heading-entreprise' className={sectionTitleStyle}>
+            <SectionTitleDot />
+            <span className='sr-only'>Informations de l&apos;</span>Entreprise
+          </h3>
+
+          <dl>
+            {offre.infoEntreprise.lien && (
+              <>
+                <dt className='sr-only'>Lien site</dt>
+                <dd className='mt-4 text-base-regular text-primary hover:text-primary_darken'>
+                  <ExternalLink
+                    href={offre.infoEntreprise.lien}
+                    label="Aller sur le site de l'entreprise"
```

Comment:
```suggestion
                    label="Site de l'entreprise"
```
suffisant, aller est redondant